Makefile.pre.in

```diff
@@ -2498,6 +2498,7 @@ TESTSUBDIRS=	idlelib/idle_test \
 		test/test_importlib/resources \
 		test/test_importlib/source \
 		test/test_inspect \
+		test/test_inspect/inspect_simple_pkg \
```

Comment:
I am not sure we really need this test-only package, because we have real ones. Is there something special about it? Like empty `__init__` or something?